app/assets/stylesheets/helpers/_header.scss

```diff
@@ -341,3 +341,42 @@
     margin-right: .5em;
   }
 }
+
+#global-bar {
```

Comment:
_I think_ this would require reworking toggling in general as this element id is assumed by the toggle module (needed by the `Hide message` link in the banner) https://github.com/alphagov/static/blob/a815620cada7ea1c65428c1c3b3ac4dbe28977bf/app/assets/javascripts/modules/toggle.js#L57

Reply:
Done in https://github.com/alphagov/static/pull/1583/commits/3fdc23e8df288099f364658e2445577790b59d67

Reply:
There's still one in there... `.show-global-bar #global-header-bar {`

Reply:
`#global-header-bar` refers to the blue bar which appears below the black GOV.UK header, it's a selector that was already part of the GOV.UK template. We're just using it to hide the blue bar and make the banner appear flush with the header.